sim_rev.h

```diff
@@ -44,17 +44,18 @@
 #define SIM_VERSION_MODE "Current"
 #endif
 
-#if defined(SIM_NEED_GIT_COMMIT_ID)
+/* Note:  Moved include to scp.c, the only place this header is
+   needed.
+*/
+#if 0 && defined(SIM_NEED_GIT_COMMIT_ID)

```

Comment:
If you want code to go away, which presumably is the case here, please delete it.  `#if 0` is fine for debugging but has no place in actually committed code.

Reply:
@pkoning2: My preference is for the code to go away, but didn't want to affront others' sensibilities with respect to legacy preservation. I'll go update the commit.